From d995ddd37df560ee8769794ceb8d5a29f0b1f308 Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@lineageos.org>
Date: Sun, 2 Dec 2018 18:57:55 +0800
Subject: [PATCH 7/8] common: Allow init to relabel I/O sched tuning nodes

 * These nodes will be labelled as sysfs_dm first on encrypted devices,
   relabelfrom permission is required to perform restorecon.

Change-Id: Id8aef9c51fec7902e59df56308cabd2a26004bed
---
 common/private/init.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/private/init.te b/common/private/init.te
index cd72a21..345c349 100644
--- a/common/private/init.te
+++ b/common/private/init.te
@@ -1,3 +1,6 @@
+allow init sysfs_dm:dir relabelfrom;
+allow init sysfs_dm:file relabelfrom;
+
 allow init {
     sysfs_io_sched_tuneable
     sysfs_livedisplay_tuneable
-- 
2.17.1

